static/50x.html

```diff
@@ -9,6 +9,9 @@
     <link type="text/css" rel="stylesheet" href="/static/css/main.css">
     <link type="text/css" rel="stylesheet" href="/static/css/account.css">
     <link rel="apple-touch-icon" href="/static/images/apple-touch-icon.png">
+    <!-- Behave more like an app when saved to home screen in iOS -->
```

Comment:
Does this actually accomplish anything on 500 pages? Seems like we only need this change for base.html

Reply:
Adding this to any page that can potentially load instead of base.html makes sense to me.